profile/README.md

```diff
@@ -0,0 +1,25 @@
+# Welcome to Pdftools GitHub Organization
+
+## 📄 Transforming Documents, One PDF at a Time
+
+At Pdftools, we specialize in providing advanced tools and services for working with PDF documents. Our mission is to build scalable, efficient, and cost-effective solutions for automating PDF-related workflows.
+
+Whether you're a developer looking for easy-to-use SDKs or a business aiming to streamline document management processes, Pdftools has you covered.
+
+## 💻 Get Started
+
+We encourage developers and contributors to explore our products, documentation and repositories, here are some useful links to get you started:
```

Comment:
@predrag-mitrovic-smallpdf do we have some interesting links to codesandbox that we could add? So that there are some links to source code?

Reply:
We have a public GitHub repository, [PDF Web Viewer Samples](https://github.com/pdf-tools/pdf-web-viewer-samples), which contains samples for the Viewer along with links to CodeSandbox for each sample.

The "getting started" sample is called [View PDF](https://codesandbox.io/p/sandbox/github/pdf-tools/pdf-web-viewer-samples/tree/4.2.0/examples/vanilla-typescript/view-pdf).

We can link to either of these, whichever you feel is more appropriate.

Reply:
To be honest feels a bit weird mentioning only Viewer and not other products, I am aware of other products not having any repos public, and that is why I was more focused on directing user to website/docs to guide them to all products for now.

Wdyt?